### PR TITLE
Add sse payload validation

### PR DESCRIPTION
**Pull request by bmaidics**

### CODE REVIEW

File: specs/binding-sse.spec/src/main/scripts/io/aklivity/zilla/specs/binding/sse/schema/sse.schema.patch.json [hunk 1]
```diff
@@ -40,6 +40,36 @@
                             {
                                 "type": "integer",
                                 "default": 2000
+                            },
+                            "paths":
```
Comment (jfallows) [inline specs/binding-sse.spec/src/main/scripts/io/aklivity/zilla/specs/binding/sse/schema/sse.schema.patch.json:44]: Not sure about `paths` ... `path`, maybe `requests` ... `paths`, or `types` ... `path`?
Let's discuss the naming.

File: runtime/binding-sse/src/test/java/io/aklivity/zilla/runtime/binding/sse/internal/streams/server/DataIT.java [hunk 2]
```diff
@@ -78,6 +78,16 @@ public void shouldReceiveMultipleMessages() throws Exception
...
+    @Test
+    @Configuration("server.validator.yaml")
+    @Specification({
+        "${net}/validation/request",
+        "${app}/validation/server" })
+    public void shouldValidateMessages() throws Exception
+    {
+        k3po.finish();
+    }
+
```
Comment (jfallows) [inline runtime/binding-sse/src/test/java/io/aklivity/zilla/runtime/binding/sse/internal/streams/server/DataIT.java:90]: We need a `shouldRejectMessages` too to prove validation is not just always allowing.

File: runtime/binding-sse/src/main/java/io/aklivity/zilla/runtime/binding/sse/config/SsePathConfig.java [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+import static java.util.function.Function.identity;
+
+import io.aklivity.zilla.runtime.engine.config.ModelConfig;
+
+public class SsePathConfig
+{
+    public final String path;
+    public final ModelConfig content;
+
```
Comment (jfallows) [inline runtime/binding-sse/src/main/java/io/aklivity/zilla/runtime/binding/sse/config/SsePathConfig.java:26]: Remove extra blank line.

**Review by jfallows — CHANGES_REQUESTED** [review]

File: runtime/binding-sse/src/test/java/io/aklivity/zilla/runtime/binding/sse/internal/streams/server/DataIT.java (hunk 2 above)
Comment (bmaidics) [reply runtime/binding-sse/src/test/java/io/aklivity/zilla/runtime/binding/sse/internal/streams/server/DataIT.java:90]: Inside this test we have an invalid data as well. I just wanted to include it in the same test, as if the payload is invalid we don't close the stream. Or maybe put it in a seperate and assert for the invalid event?

File: specs/binding-sse.spec/src/main/scripts/io/aklivity/zilla/specs/binding/sse/schema/sse.schema.patch.json (hunk 1 above)
Comment (bmaidics) [reply specs/binding-sse.spec/src/main/scripts/io/aklivity/zilla/specs/binding/sse/schema/sse.schema.patch.json:44]: Yeah, I was thinking about this as well a lot, but couldn't come up with a better one. I think the `requests` ... `path` makes more sense.

File: runtime/binding-sse/src/test/java/io/aklivity/zilla/runtime/binding/sse/internal/streams/server/DataIT.java (hunk 2 above)
Comment (jfallows) [reply runtime/binding-sse/src/test/java/io/aklivity/zilla/runtime/binding/sse/internal/streams/server/DataIT.java:90]: Yes, agreed. Please separate the positive validation and negative validation scenarios.

File: specs/binding-sse.spec/src/main/scripts/io/aklivity/zilla/specs/binding/sse/schema/sse.schema.patch.json (hunk 1 above)
Comment (jfallows) [reply specs/binding-sse.spec/src/main/scripts/io/aklivity/zilla/specs/binding/sse/schema/sse.schema.patch.json:44]: Agreed, let's go with `requests` then, I was leaning towards that one too. 😄 

The idea of `types` is too overloaded already, it's why we went with `model` for the validation.

**Review by jfallows — APPROVED** [review]

**Pull request merged**